packages/instantsearch.js/src/lib/server.ts

```diff
@@ -37,17 +57,27 @@ export function waitForResults(search: InstantSearch): Promise<void> {
 /**
  * Walks the InstantSearch root index to construct the initial results.
  */
-export function getInitialResults(rootIndex: IndexWidget): InitialResults {
+export function getInitialResults(
+  rootIndex: IndexWidget,
+  /**
+   * Search parameters sent to the search client, usually
```

Comment:
usually?

Reply:
There's a follow-up below 😅

> Search parameters sent to the search client, usually returned by `waitForResults()`.

Reply:
I mean when is it not returned from waitForResults?

Reply:
Ah I get it, currently, never. I'll remove this word.